api/controllers/track.js

```diff
@@ -49,3 +90,92 @@ export async function byTrackID(req, res) {
     }
   }
 }
+
+/**
+ * function: getAlbumQuery
+ * creates query and parameters for an album query to the database
+ * currently does not work with prepared statements
+ *
+ * @param {String} albumID - string for fuzzy search
+ * @param {String} artistID - tag for fuzzy search
+ * @param {Int} name - parentID for exact search
+ * @param {String} length - updated after given date
```

Comment:
should you break this up into `longerThan` and `shorterThan`? Can come back to this since that will require some validation making sure longerThan is less than shorterThan.

Reply:
ok- lets come back to this